mopidy/models/fields.py

```diff
@@ -72,20 +73,41 @@ def __init__(self, default=None):
 
 
 class Date(String):
+    """
+    :class:`Field` for storing ISO 8601 dates as a string.
+
+    Supported formats are ``YYYY-MM-DD``, ``YYYY-MM`` and ``YYYY``, currently
+    not validated.
+
+    :param default: default value for field
+    """
     pass  # TODO: make this check for YYYY-MM-DD, YYYY-MM, YYYY using strptime.
 
 
 class Identifier(String):
+    """
+    :class:`Field` for storing ASCII values such as GUIDs or other identifiers.
+
+    Values will be interned.
+
+    :param default: default value for field
+    """
     def validate(self, value):
         return intern(str(super(Identifier, self).validate(value)))
 
 
 class URI(Identifier):
+    """
+    :class`Field` for storing URIs
+
+    Values will be interned, currently not validated.
+
+    :param default: default value for field
+    """
     pass  # TODO: validate URIs?
 
 
 class Integer(Field):
-
```

Comment:
True, but we haven't tried that on the code yet? Just the auto-pep-8 thing as far as I remember.